packages/core/remotes/impl/outreach/index.ts

```
@@ -1478,7 +1478,17 @@ class OutreachClient extends AbstractEngagementRemoteClient {
         headers: this.getAuthHeadersForPassthroughRequest(),
       }
     );
-    return response.data.data.id.toString();
+
+    const sequenceId = response.data.data.id.toString();
+    // Should we do this in parallel? Are there rate limit concerns?
+    for (const [index, step] of (params.steps ?? []).entries()) {
+      await this.createSequenceStep({
+        ...step,
+        order: step.order ?? index + 1,
```

Instead of this can you fail the request for Outreach if `order` is not specified?

---

It's actually the opposite, `order` should be omitted when creating steps together with sequence. I'm modifying to ignoring rather than failing it.